src/Compilers/CSharp/Test/Semantic/Semantics/UsingStatementTests.cs

```diff
@@ -1784,6 +1781,32 @@ void M()
                 Diagnostic(ErrorCode.WRN_PossibleMistakenNullStatement, ";").WithLocation(11, 30));
         }
 
+        [Fact]
+        public void MissingIDisposable_NoLocal()
+        {
+            var source = @"
+namespace System
+{
+    public class Object { }
+    public class Void { }
+}
+class C
+{
+    void M()
+    {
+        using (null);
```

Comment:
>; [](http://example.com/codeflow?start=20&length=1)

Consider using `{ }` to avoid the warning.

Reply:
This is intended to be like test above, just without the declared local